win32ss/user/ntuser/winpos.c

```diff
@@ -1393,9 +1393,13 @@ WinPosDoOwnedPopups(PWND Window, HWND hWndInsertAfter)
 
                if (List[i] == Owner)
                {
-                  if (i > 0) hWndInsertAfter = List[i-1];
-                  else hWndInsertAfter = topmost ? HWND_TOPMOST : HWND_TOP;
-                  break;
+                   if (i > 0)
+                   {
+                       if (List[i-1] != Window->head.h)  
+                           hWndInsertAfter = List[i-1];
```

Comment:
```suggestion
                       if (List[i - 1] != Window->head.h)  
                           hWndInsertAfter = List[i - 1];
```

Reply:
Changed. Thanks.

Reply:
It doesn't look like so...

Reply:
They came back in a following commit. Now fixed again. Thanks.